assets/custom.js

```diff
@@ -76,11 +76,28 @@ function resetForm() {
 
 //GET cart object and append data to 'cart' icon in navbar
 function updateCart() {
+    $.ajax({
+        url: '/cart'
+      }).done(function(data) {
+        //console.log('updateCart: data:', data)
+        //console.log('data html:', $(data).find('.mini-cart').html())
+        $('.mini-cart__empty').remove();
+        $('.mini-cart__list').html($(data).find('.mini-cart__list').html()); //ISSUE
```

Comment:
is this still an issue or just leftover comment?

Reply:
Left over comment. Erased now

Reply:
Forgot about adding the animation. Added as well.